### DISCUSSION TITLE
gatsby-source-wordpress npm run docker-start fails

### DISCUSSION THREAD

**Discussion by lnikell** [gatsbyjs/gatsby#30871] [Help]
## Description

I'm trying to make some changes to `gatsby-source-wordpress`, however, I did not find a guide regarding how to properly contribute to that package. I tried to figure out based on the repository that you use docker with wordpress to run a test environment to experiment with the plugin but when I run 'npm run docker-start' it fails with the following error
![image](https://user-images.githubusercontent.com/2697570/114695793-4a87f000-9d1c-11eb-8079-b543c029b0b3.png)

It would be great to add to the documentation which files should be created in order to contribute to the plugin. I probably will fix that error later on, however it would be great to get information, what those missing files should actually contain.

```
      - ./test-site/.env.test
      - ./test-site/.env.plugins
```
### Steps to reproduce

1. Clone latest gatsby repository and use npm install
2. Go to gatsby-source-wordpress and use `npm install`
3. Run `npm run docker-start`

### Expected result

Should not be an error

### Actual result

Error from the description section

### Environment

```
  System:
    OS: macOS 11.0.1
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.13.0 - /usr/local/bin/node
    Yarn: 1.21.0 - /usr/local/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 89.0.4389.128
    Firefox: 76.0.1
    Safari: 14.0.1
  npmPackages:
    gatsby-core-utils: ^2.4.0-next.0 => 2.4.0-next.0
    gatsby-image: ^3.4.0-next.0 => 3.4.0-next.0
    gatsby-plugin-catch-links: ^3.4.0-next.0 => 3.4.0-next.0
    gatsby-source-filesystem: ^3.4.0-next.0 => 3.4.0-next.0
  npmGlobalPackages:
    gatsby-cli: 2.8.23
```

**Comment by lnikell** [comment]
I was able to move forward so it seems in .env.plugins should be something like this:
```
WPGRAPHQL_VERSION=v1.3.5
WPGATSBY_VERSION=v1.0.8
WPGRAPHQL_ACF_VERSION=v0.4.0
WPGRAPHQL_CPT_UI_VERSION=v1.1
ACF_VERSION=master
```
I'm starting to think that nobody is using that docker configuration for development at all, since the seed/backup.sql has a siteurl and home defined as "https://devgatsbyint.wpengine.com"

**Comment by LekoArts** [comment]
cc @TylerBarnes

**Comment by TylerBarnes** [comment]
@lnikell docker was temporarily disabled when we moved from the gatsby-source-wordpress-experimental repo. I just opened a PR which restores our docker setup for int tests https://github.com/gatsbyjs/gatsby/pull/30938

**Reply by lnikell** [reply]
thanks, could you give me some recommendations about how to develop this plugin locally, since it requires some dependencies that are not exist in package json and when i try to follow all suggested by Gatsby practices for building/contribution it just does not work.

**Reply by TylerBarnes** [reply]
@lnikell for sure! I'll update that PR ☝️ with a contributing section and let you know when it's there

**Reply by TylerBarnes** [reply]
@lnikell I added docs here https://github.com/gatsbyjs/gatsby/pull/30938/commits/3417249ad16416f346979814cdb5ec43d4a7157d
